Code_Exercises/Exercise_01_Compiling_with_SYCL/solution.cpp

```diff
@@ -9,9 +9,14 @@
 */
 
 #define CATCH_CONFIG_MAIN
-#include <SYCL/sycl.hpp>
 #include <catch2/catch.hpp>
 
+#if defined(SYCL_LANGUAGE_VERSION) && defined(__INTEL_LLVM_COMPILER)
+#include <CL/sycl.hpp>
+#else
+#include <SYCL/sycl.hpp>
```

Comment:
Out of curiosity, is there any reason why we use `SYCL/sycl.hpp` instead of the SYCL 2020 final `sycl/sycl.hpp`? (For hipSYCL it doesn't matter as we support both)

Reply:
It should be `sycl/sycl.hpp`

Reply:
`sycl/sycl.hpp` is generally fine for us. One nitpick is that because we still support `SYCL/sycl.hpp`, including `sycl/sycl.hpp` on Windows will in some cases trigger `-Wnonportable-include-path`.